homeassistant/components/lovelace/__init__.py

```diff
@@ -246,3 +247,8 @@ def _register_panel(hass, url_path, mode, config, update):
         kwargs["sidebar_icon"] = config.get(CONF_ICON, DEFAULT_ICON)
 
     frontend.async_register_built_in_panel(hass, DOMAIN, **kwargs)
+
+
+async def system_health_info(hass):
```

Comment:
Why proxy it?

`# pylint: disable=unused-import` in the import should be enough, right?

Reply:
Good morning! :coffee: 
You are absolutely correct sir!